common/adaptive-bitrate.md

```diff
@@ -0,0 +1,110 @@
+
+### Publisher-side configuration
+
+Simulcast configuration is opt-in and can be enabled on the role's template. The role's publish video quality determines video quality layers on simulcast. For example, a role configured to publish at 720p can simulcast 180p, 360p and 720p layers.
+
+| Video publish quality | Possible simulcast layers |
+|-----------------------|---------------------------|
+| 1080p                 | 270p, 540p, 1080p         |
+| 720p                  | 180p, 360p, 720p          |
+| 480p                  | 240p, 480p                |
+| 360p                  | 180p, 360p                |
+| 240p                  | 240p                      |
+| 180p                  | 180p                      |
+
+#### Enable via dashboard
+
+_This feature is in limited availability._
+
+Enable "can publish simulcast" on the template page for a particular role. You can also specify how many video quality layers will be simultaneously published by peers of this role. The peer will publish these layers assuming network bandwidth permits.
+
+![Simulcast configuration](/guides/simulcast-on-dashboard.png)
+
+#### Enable via API
+
+Update role configuration using the [server-side API](/docs/server-side/v2/policy/create-update-role). The simulcast config payload can include 2 or 3 layers that scale down the selected publish layer.
+
+In the example below, the role is configured to publish 720p with 3 simulcast layers:
+
+* `f` for full with scale down factor of 1 (= 720p)
+* `h` for half with scale down factor of 2 (= 360p)
+* `q` for quarter with scale down factor of 4 (= 180p)
+
+```js
+{
+    "publishParams": {
+        ...
+        "simulcast": {
+            "video": {
+                "layers": [
+                    {
+                        "rid": "f",
+                        "scaleResolutionDownBy": 1,
+                        "maxBitrate": 700,
+                        "maxFramerate": 30
+                    },
+                    {
+                        "rid": "h",
+                        "scaleResolutionDownBy": 2,
+                        "maxBitrate": 250,
+                        "maxFramerate": 30
+                    },
+                    {
+                        "rid": "q",
+                        "scaleResolutionDownBy": 4,
+                        "maxBitrate": 100,
+                        "maxFramerate": 30
+                    }
+                ]
+            },
+            "screen": {}
+        }
+    }
+}
+```
+
+### Subscribe-side behavior
+
+<br/>
+
+#### Manual layer selection
+
+The 100ms client-side SDKs provide methods to set a preferred quality layer for a remote peer's video track. See docs for your preferred platform:
+
+* [JavaScript](/docs/javascript/v2/advanced-features/simulcast)
+* [iOS](/docs/ios/v2/advanced-features/simulcast)
+* [Android](/docs/android/v2/advanced-features/simulcast)
```

Comment:
no docs for API 🤔 ?

Reply:
API docs are not relevant for the subscriber side. i have linked them on the publisher side for template update